gobblin-metrics-libs/gobblin-metrics-base/src/main/avro/GaaSObservabilityEventV0.avsc

```diff
@@ -0,0 +1,135 @@
+{
+  "type": "record",
+  "name": "GaaSObservabilityEventExperimental",
+  "namespace" : "org.apache.gobblin.metrics",
+  "doc": "An experimental feature for GaaS to emit events during and after a job is executed.",
```

Comment:
nit: yes, the *feature* is experimental, but the most experimental thing about this schema is it's *format*